include/quicr/quicr_server.h

```diff
@@ -145,6 +145,22 @@ class ServerDelegate
                            bytes&& data)
   {
   }
+
+  /**
+   * @brief Unsubscribe callback method
+   *
+   * @details Called for each unsubscribe message
+   *
+   * @param quicr_namespace          QuicR name/len
+   * @param subscriber_id            Subscriber ID connection/transport that
+   *                                 sent the message
+   * @param auth_token               Auth token to verify if value
+   */
+  virtual void onUnsubscribe(const quicr::Namespace& quicr_namespace,
+                             const uint64_t& subscriber_id,
+                             const std::string& auth_token)
+  {
+  }
```

Comment:
Should this be pure virtual? If not, why not? If still not, should we not declare the param names, or else mark them as [[maybe_unused]] to reduce compiler warnings?

Reply:
I only ask because we're already drowning in similar warnings already.

Reply:
It's optional, so not pure virtual. If not implemented, then it's doesn't do anything. 

Regarding compiler warnings, they are many. For me, they are all "unused parameter".  We can address those in a different PR over all the repos.